robotd/devices_base.py

```diff
@@ -11,12 +11,15 @@ class BoardMeta(type):
     """
 
     BOARDS = []
+    STARTUP_BOARDS = []
 
     def __new__(cls, name, bases, dict):
         """Instantiate subclass."""
         cls = type.__new__(cls, name, bases, dict)
         if hasattr(cls, 'lookup_keys') and cls.enabled:
             BoardMeta.BOARDS.append(cls)
+        if hasattr(cls, 'create_on_startup'):
+            BoardMeta.STARTUP_BOARDS.append(cls)
```

Comment:
Is it better to do this rather than leaving it to users of `BOARDS` to check the flag when iterating over it?